src/new/wallet.js

```diff
@@ -566,7 +570,12 @@ class HathorWallet extends EventEmitter {
    **/
   handleWebsocketMsg(wsData) {
     if (wsData.type === 'wallet:address_history') {
-      this.onNewTx(wsData);
+      if ([HathorWallet.PROCESSING, HathorWallet.SYNCING].includes(this.state)) {
```

Comment:
I feel it's better to check whether state is not ready. So we can safely add other states in the future.

Reply:
Done

Reply:
Thanks!